index.html

```diff
@@ -67,5 +67,5 @@
 talk:
     title: Talk to Us!
     subtitle: We would love to hear from you, how you are using Kubernetes, and what we can do to make it better.
-    text: Our IRC channel is the best way to contact our engineers and share your ideas with them. They hang out at <a href="http://webchat.freenode.net/?channels=google-containers" onclick="trackOutboundLink('http://webchat.freenode.net/?channels=google-containers'); return false;">#google-containers</a> on Freenode. Our user forum on <a href="http://stackoverflow.com/questions/tagged/kubernetes" onclick="trackOutboundLink('http://stackoverflow.com/questions/tagged/kubernetes'); return false;">stackoverflow</a> is a great place to go for community support<a href="http://kubernetes.io/v1.1" onclick="trackOutboundLink('http://kubernetes.io/v1.1'); return false;">.</a>
+    text: Our Slack channel is the best way to contact our engineers and share your ideas with them.They hang out at <a href="https://kubernetes.slack.com" onclick="trackOutboundLink('https://kubernetes.slack.com'); return false;">#kubernetes-users</a> on Slack. Our user forum on <a href="http://stackoverflow.com/questions/tagged/kubernetes" onclick="trackOutboundLink('http://stackoverflow.com/questions/tagged/kubernetes'); return false;">stackoverflow</a> is a great place to go for community support<a href="http://kubernetes.io/v1.1" onclick="trackOutboundLink('http://kubernetes.io/v1.1'); return false;">.</a>
```

Comment:
Same comment about URL.

Also, you accidentally deleted the space after the first period.